sdks/python/apache_beam/coders/row_coder.py

```diff
@@ -162,6 +164,8 @@ def _nonnull_coder_from_type(field_type):
       return BooleanCoder()
     elif field_type.atomic_type == schema_pb2.BYTES:
       return BytesCoder()
+    elif field_type.atomic_type == schema_pb2.BYTE:
```

Comment:
Same here (according to #19815)

Reply:
removed. Thanks!

Reply:
- nvm - Byte seems to be okay. Should I leave it as is?

Reply:
It seems OK because when BytesCoder encoding/decoding single byte with nested=False is the same as Java's ByteCoder (write one bytes); but if the stream has other data after this byte it is not the case.

Reply:
@pabloem commented here

Reply:
Thanks for raising this @Abacn :)